.travis.yml

```diff
@@ -10,7 +10,7 @@ cache:
   - ./texlive/texlive
 
 script:
-- latexmk -pdf --outdir=build --auxdir=build --halt-on-error skript.tex
+- latexmk -pdf --outdir=build --auxdir=build --interaction=nonstopmode --halt-on-error skript.tex
 - mv build/skript.pdf page/
```

Comment:
This is necessary so pdflatex will not wait for user input on missing files but actually hold. The flag --halt-on-error does not imply this behaviour on its own.